docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -347,15 +347,15 @@ Formatting is influenced by the properties of the current <xref:System.Globaliza
 
```

Comment:
@pgovind Is there a special format specifier to round-trip a Half value like there is for Double and Single (see line 288)?

Reply:
Also, I noticed that Half has properties whereas Single and Double have fields. Is that intentional?

Reply:
This is intentional. `double` and `single` are runtime primitives and have special encoding in metadata for constants. `Half` is functionally a "user-defined" type and so cannot declare constants today.

Reply:
> Is there a special format specifier to round-trip a Half value like there is for Double and Single (see line 288)

`G5` for Half

Reply:
Since `Half` is only supported on .NET Core, `R` is the right thing to use here and will always do the correct thing.

Reply:
We only need to list `G9` and `G17` for float/double because `R` had bugs on full framework.

Reply:
Ah that explains why. I was wondering what cases `R` gave wrong results for.